where-to-travel/src/main/webapp/script.js

```diff
@@ -348,17 +367,33 @@ function addLoginButtons() {
  */
 function addUserDash() {
   const dashElement = $(getUserDashHtml(user));
-  $(dashElement[2]).click(function () {
+  // reverse the boolean control on click
```

Comment:
For my understanding, could you clarify why we reverse the boolean control here?

Reply:
That was an old comment, I removed it in 54cc36c